bitrise.yml

```diff
@@ -0,0 +1,195 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - BITRISE_STEP_ID: post-jira-comment
+  - BITRISE_STEP_VERSION: "0.9.0"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-steplib/steps-post-jira-comment-with-build-details
+  - MY_bitrise-steplib_REPO_FORK_GIT_URL: $MY_bitrise-steplib_REPO_FORK_GIT_URL
+
+  - BITRISE_JIRA_USER_NAME: $BITRISE_JIRA_USER_NAME
+  - BITRISE_JIRA_API_TOKEN: $BITRISE_JIRA_API_TOKEN
+  - BITRISE_JIRA_BASE_URL: $BITRISE_JIRA_BASE_URL
+  - BITRISE_JIRA_ISSUE_KEYS: $BITRISE_JIRA_ISSUE_KEYS
+
+workflows:
+  dep-update:
+    title: Dep update
+    description: |
+      Used for updating bitrise dependencies with dep
+    steps:
+    - script:
+        title: Dependency update
+        inputs:
+        - content: |-
+            #!/bin/bash
+            set -ex
+            go get -u -v github.com/golang/dep/cmd/dep
+            dep ensure -v
+            dep ensure -v -update
+
+  go-test:
```

Comment:
Please use the new go steps: https://github.com/godrei/steps-generate-changelog/blob/master/bitrise.yml#L14